docs/knobs/overview.mdx

```diff
@@ -78,4 +78,5 @@ Widgetbook's UI:
 | doubleOrNull.slider | `double?` |
 | double.input        | `num`     |
 | doubleOrNull.input  | `num?`    |
+| listOrNull          | `T?`      |
 | list                | `T`       |
```

Comment:
```suggestion
| list                | `T`       |
| listOrNull          | `T?`      |
```

Nullable knob options are below the non-null options.


![Screenshot 2023-08-15 at 15 27 33](https://github.com/widgetbook/widgetbook/assets/48603081/456dde4c-4f3f-4640-b584-bf0778997134)